### PR TITLE
Adding Maistra Molecule Testing with Maistra UBI8 1.0.0

### PR DESCRIPTION
**Pull request by gbaufake**
> Note 
The ubi8 images are not yet available on docker.io but they are same as specified in https://github.com/Maistra/istio-operator/blob/maistra-1.0/deploy/smcp-templates/maistra

### CODE REVIEW

File: operator/molecule/README.md [hunk 1]
```diff
@@ -22,9 +22,12 @@ If you want to run without the destroy, you can run with `molecule test --destro
...
 
 
 For all the scenarios use `molecule test --all`.
 
-For debugging use `molecule --debug  test` (it needs to be before the verb - test, prepare, destroy)
\ No newline at end of file
+For debugging use `molecule --debug  test` (it needs to be before the verb - test, prepare, destroy)
+
+
+##
```
Comment (jmazzitelli) [inline operator/molecule/README.md]: what are the three lines here (two empty lines and one line with just `##`)

File: operator/molecule/requirements.txt [hunk 2]
```diff
@@ -4,5 +4,7 @@ molecule
 openshift
 jmespath
 junit-xml
+kubernetes
 pyasn1>=0.4.6
 python-ldap
+docker
```
Comment (jmazzitelli) [inline operator/molecule/requirements.txt]: Don't forget to make these same changes to our molecule Dockerfile - some of us don't have python running correctly on our local box and use the molecule docker image for running these tests :)

So add these to this [Dockerfile](https://github.com/kiali/kiali/blob/master/operator/molecule/docker/Dockerfile#L3) and test it by re-building the new docker image and run the maistra-e2e scenario:

```
MOLECULE_SCENARIO=maistra-e2e make molecule-docker-build molecule-test
```

**Comment by jmazzitelli** [comment]
I see @mattmahoneyrh is listed at a reviewer too - I'll wait for him to say this is good. I'd like for someone to tell me this final version of the test in this PR runs successfully for them.

**Comment by gbaufake** [comment]
Fixes #1650

**Comment by gbaufake** [comment]
@mattmahoneyrh 
It is good to go now for UBI8 Image.

**Pull request merged**